src/Github/Api/GraphQL/RunGraphQLQuery.php

```diff
@@ -56,15 +54,10 @@ public function __invoke(
             ->getBody()
             ->__toString();
 
-        Assert::same($response->getStatusCode(), 200);
-
-        $responseData = json_decode($responseBody, true);
-
-        Assert::isMap($responseData);
-        Assert::keyNotExists($responseData, 'errors');
-        Assert::keyExists($responseData, 'data');
-        Assert::isArray($responseData['data']);
+        Type\literal_scalar(200)->assert($response->getStatusCode());
```

Comment:
`Assert::keyNotExists($responseData, 'errors');` <- this assertion was lost

/cc @Ocramius